catalog.html

```diff
@@ -0,0 +1,319 @@
+                    </ul>
+                </fieldset>
+            </form>
+        </section>
+
+        <section class="price">
+            <fieldset class="price-selection">
+                <legend class="price-selection-title">Стоимость, ₽:</legend>
+
+                <button class="button" type="submit">ПРИМЕНИТЬ</button>
+                <button class="button" type="reset">Сбросить</button>
+            </fieldset>
+        </section>
+
+        <section class="catalog">
+            <div class="sort-catalog">
+                <div class="search-result">
+                    НАЙДЕНО ГОСТИНИЦ: 38
+                </div>
+                <div class="select-catalog">
+                    <select class="select-catalog-price" aria-label="Сортировка">
+                        <option value="cheap">Сначала дешёвые</option>
+                    </select>
+                </div>
+                <div class="view">
+                    <img class="view-icon" src="#" alt="Таблица">
+                    <img class="view-icon" src="#" alt="Слайд-шоу">
+                    <img class="view-icon" src="#" alt="Список">
+                </div>
+            </div>
+        </section>
+
+        <section class="catalog">
+            <ul class="catalog-list">
+                <li class="catalog-item">
+                  <img class="catalog-hotel" src="#" alt="Гостиница Amara Resort & Spa">
+                  <h2 class="catalog-title">Amara Resort & Spa</h2>
+                  <div>
+                    <span class="catalog-type">Гостиница</span>
+                    <span class="catalog-price">От 4000 ₽</span>
+                  </div>
+                  <div class="catalog-price-button">
+                    <button class="price-button-info">Подробнее</button>
+                    <button class="price-button-favourites">В избранное</button>
+                  </div>
+                  <div class="catalog-rating">
+                    <div class="rating-stars">
+                      <img src="#" alt="Рейтинговая звезда">
+                      <img src="#" alt="Рейтинговая звезда">
+                      <img src="#" alt="Рейтинговая звезда">
+                      <img src="#" alt="Рейтинговая звезда">
+                    </div>
+                    <div class="rating-description">Рейтинг: 8,5</div>
+                  </div>
+                </li>
+
+                <li class="catalog-item">
+                  <img class="catalog-hotel" src="#" alt="Гостиница Villas at Poco Diablo">
+                  <h2 class="catalog-title">Villas at Poco Diablo</h2>
+                  <div>
+                    <span class="catalog-type">Гостиница</span>
+                    <span class="catalog-price">От 5000 ₽</span>
+                  </div>
+                  <div class="catalog-price-button">
+                    <button class="price-button-info">Подробнее</button>
+                    <button class="price-button-favourites">В избранное</button>
```

Comment:
```suggestion
                    <button class="price-button-favourites">В избранном</button>
```
Так как это другое состояние.